packages/core-utils/src/watcher.ts

```diff
@@ -61,10 +61,11 @@ export class Watcher {
     for (const log of receipt.logs) {
       if (
         log.address === layer.messengerAddress &&
-        log.topics[0] === ethers.utils.id('SentMessage(bytes)')
+        log.topics[0] ===
+          ethers.utils.id('SentMessage(address,address,bytes,uint256,uint256)')
```

Comment:
This could be a pre-computed global variable, it doesn't need to be hashed each time